app/src/main/java/org/mozilla/tv/firefox/ext/Locale.kt

```diff
@@ -1,3 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
nit: can we add comments on the duplicated logic too, as per my response in the original PR: https://github.com/mozilla-mobile/firefox-tv/pull/2545/files#r303647320 I just added it so don't worry, you didn't miss it. 😛